feat: Issue form templates for easier and better issue creation

Palanikannan1437 opened the pull request:
## Suggested Changes

This might be really useful and convenient for the community to create new issues easily due to the simple form format and detailed instructions!!

Added 2 Issue form templates
1. Bug Report Template
<img width="479" alt="image" src="https://user-images.githubusercontent.com/73993394/188366223-5dff459d-8911-484c-967a-6a7e6a0928be.png">

---

2. Feature Request Template
<img width="837" alt="image" src="https://user-images.githubusercontent.com/73993394/188366444-9a4661fd-e897-49ff-8d41-15c08befc882.png">

jeanfbrito commented:
Hey @Palanikannan1437 I changed some things and removed the feature-request file for now.

Palanikannan1437 commented:
> Hey @Palanikannan1437 I changed some things and removed the feature-request file for now.

Sure thing @jeanfbrito, the changes do make a lot of sense! Thank you so much for reviewing it😄

Also, was their any specific reason for removing the feature-request template for now?
